tests/api.c

```diff
@@ -21201,12 +21218,22 @@ static int test_wc_ecc_shared_secret (void)
             ret = wc_ecc_init(&pubKey);
         }
     }
+
+#if defined(HAVE_ECC) && !defined(NO_ECC256)
+    if (ret == 0) {
+      ret = wc_ecc_import_raw(&key, qx, qy, d, curveName);
+    }
+    if (ret == 0) {
+      ret = wc_ecc_import_raw(&pubKey, qx, qy, d, curveName);
```

Comment:
Don’t pass “d” in here and you will have a public only key as it is named.

Reply:
Thanks. Done